app/services/trading_strategies.py

```
@@ -4215,6 +4614,304 @@ async def _estimate_daily_volatility(self, symbol: str) -> float:
+            for pos in positions:
+                weight = pos.get('value', 0) / total_value
+                pos_volatility = pos.get('volatility', 0.05)
+                portfolio_volatility += (weight * pos_volatility) ** 2
+            
+            portfolio_volatility = portfolio_volatility ** 0.5
+            
+            # VaR calculation (95% confidence level)
+            var_1d = total_value * portfolio_volatility * 1.65  # 95% confidence
+            var_1w = var_1d * (7 ** 0.5)  # Weekly VaR
+            
+            return {"var_1d": var_1d, "var_1w": var_1w}
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)